x-pack/plugins/monitoring/server/lib/elasticsearch/ccr.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
+
+export function handleResponse(response) {
+  return get(response.defaults, 'xpack.ccr.enabled', true);
```

Comment:
Nit/Question: what's the difference/advantage of using this construction:

```js
get(response.defaults, 'xpack.ccr.enabled', true);
```

Over this one?

```js
get(response, 'defaults.xpack.ccr.enabled', true);
```

Reply:
The call should be a tiny bit faster as we can ensure that `defaults` exists on the response object, but we can't be sure that `xpack` exists off `defaults`

Reply:
Ah, make sense, thanks.